README.md

```diff
@@ -54,18 +58,68 @@ npx twilio-run
 
 ## Usage
 
+Check out the [commands](#commands) for in depth usage, but here are some things you will want to know:
```

Comment:
Could we link somewhere to the rest of the docs on twilio.com/docs/labs/serverless-toolkit? We'll probably have Maria soon update more of the docs there so it would be useful to have those docs over there as well.

Reply:
There is a link to the Twilio docs above, under the headline "About". I have started more docs here though, as the repo is one place that people specifically look for documentation. Being able to get the answers here without an additional click is a better experience (not that twilio.com shouldn't host more in depth documentation about this, which I welcome).